lms/djangoapps/courseware/views/views.py

```diff
@@ -318,17 +307,17 @@ def get_last_accessed_courseware(course, request, user):
         dates_fragment = None
 
         if request.user.is_authenticated():
+            # TODO: LEARNER-611: Remove enable_course_home_improvements
             if SelfPacedConfiguration.current().enable_course_home_improvements:
+                # Shared code with the new Course Home (DONE)
                 dates_fragment = CourseDatesFragmentView().render_to_fragment(request, course_id=course_id)
 
         # This local import is due to the circularity of lms and openedx references.
         # This may be resolved by using stevedore to allow web fragments to be used
         # as plugins, and to avoid the direct import.
         from openedx.features.course_experience.views.course_reviews import CourseReviewsModuleFragmentView
 
-        # Decide whether or not to show the reviews link in the course tools bar
-        show_reviews_link = CourseReviewsModuleFragmentView.is_configured()
```

Comment:
Reviewers: I think this was no longer used once we switched to `CourseToolsPluginManager.get_enabled_course_tools`.